Ryujinx/Ui/ApplicationLibrary.cs

```diff
@@ -115,8 +118,8 @@ public static void Init(List<string> AppDirs, Keyset keySet, SystemState.TitleLa
                             }
 
                             // Creates NACP class from the NACP file
-                            IFile controlNacp = controlFs.OpenFile("/control.nacp", OpenMode.Read);
-                            Nacp  controlData = new Nacp(controlNacp.AsStream());
+                            controlFs.OpenFile(out IFile controlNacpFile, "/control.nacp", OpenMode.Read).ThrowIfFailure();
+                            Nacp  controlData = new Nacp(controlNacpFile.AsStream());
```

Comment:
```suggestion
                            Nacp controlData = new Nacp(controlNacpFile.AsStream());
```
and add an empty line before.